src/main/java/frc/robot/util/RobotMath.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.util;
+
+public class RobotMath {
+    public static double powerCurve(double x) {
+        // https://www.desmos.com/calculator/g07ukjj7bl
+        double curve = (0.5D * (Math.atan(Math.PI * (Math.abs(x) - 0.5D)))) + 0.5D;
+        return Math.copySign(curve, x);
+    }
+
+    public static void normalize(double[] array) {
+        double max = Math.abs(array[0]);
+        boolean normFlag = max > 1;
+
+        for (int i = 1; i < array.length; i++) {

```

Comment:
Why not iterate over the whole array if we basically treat `array[0]` the same as the rest of the items? You could initialize `max` as 0, `normFlag` as `false`, then make this loop much cleaner as `for (double reasonableName : array) {`.